ql/src/test/queries/clientpositive/rename_translated_external_table.q

```diff
@@ -0,0 +1,18 @@
+set metastore.metadata.transformer.class=org.apache.hadoop.hive.metastore.MetastoreDefaultTransformer;
+set hive.fetch.task.conversion=none;
+set hive.compute.query.using.stats=false;
+
+create database if not exists tmp;
+
+use tmp;
+
+create table b(s string) stored as parquet location 'file:${system:test.tmp.dir}/tmp.db/some_location';
+
+desc formatted b;
```

Comment:
Output of the query does not show the details about the location, so I think it better to do like this:
```sql
insert into table b values('a'),('b'),('c');
select * from b;
```
after renaming b to bb, then
```
select * from bb;
-- new table and location should be available after rename
create table b(s string) stored as parquet location 'file:${system:test.tmp.dir}/tmp.db/some_location';
select * from b;  -- the result should be empty
```

Reply:
ack